packages/babel-plugin-transform-remove-unused-catch-binding/test/fixtures/remove-unused-catch-binding/try-catch-block-unused-binding-new-variable/actual.js

```diff
@@ -0,0 +1,5 @@
+try {
+  throw 0;
+} catch (e) {
+  let e = new TypeError('New variable declaration, initialization, and assignment; the catch binding is not being referenced and can be removed.');
```

Comment:
Yeah. This is fixed in #5980 . and that `let e` will throw an error now. Can you rebase your branch to `7.0`?